test/e2e/utils/ui/surveyFormPage.js

```diff
@@ -0,0 +1,74 @@
+import {
+  click,
+  getElement,
+  toRightOf,
+  writeIntoTextBox,
+  button,
+  clearTextBox,
+  expectExists,
+  hover,
+  expectToBe,
+} from '../api'
+import { waitForLoader } from './loader'
+import { expectSurveyFormLoaded } from './surveyForm'
+
+const selectors = {
+  name: () => toRightOf('Name'),
+  label: () => toRightOf('Label'),
+  language: () => toRightOf('Language(s)'),
+  multiple: () => toRightOf('Multiple'),
+}
+
+export const editPage = async ({ values }) => {
```

Comment:
I would make names more explicit. page is too generic; a page can exists in other ui vies as well.
this name and below methods can include surveyForm. e.g. editSurveyFormPage.

Reply:
I will change that, I used to do the file like a library and then use the functions like methods:

surveyForm.addPage, surveyForm.editPage, 

I will rename the functions as you propose.